contracts/registry/ClaimTopicsRegistry.sol

```diff
@@ -2,6 +2,7 @@ pragma solidity ^0.6.0;
 
 import "../registry/IClaimTopicsRegistry.sol";
 import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "../destroyable/Destroyable.sol";
 
 contract ClaimTopicsRegistry is IClaimTopicsRegistry, Ownable {
```

Comment:
Inheriting ownable is not required in claim topics registry if we are already inheriting it in Destroyable.

Reply:
indeed